src/Mutation/PostObjectUpdate.php

```diff
@@ -105,6 +113,10 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
 				throw new UserError( sprintf( __( 'Sorry, you are not allowed to update %1$s as this user.', 'wp-graphql' ), $post_type_object->graphql_plural_name ) );
 			}
 
+			/**
```

Comment:
It's a bit unclear what this comment is for. Is this just a high level grouping comment for all the code that follows it?

Reply:
🤔  ya. . .I think I moved the code I was working on up to lines 98-105 but left a comment I had started writing and didn't catch it before committing. . .lol

I'll remove this comment.